src/test/java/com/google/firebase/remoteconfig/TemplateTest.java

```diff
@@ -132,5 +132,12 @@ public void testEquality() {
     assertNotEquals(templateThree, templateFive);
     assertNotEquals(templateThree, templateSeven);
     assertNotEquals(templateFive, templateSeven);
+
+    final Template templateNine = new Template()
```

Comment:
It's ok to define some of these as class level constants. For example templateNine can be a class-level constant, and templateTen can be initialized in the corresponding test case as a local var. Same goes for others.

Reply:
I see what you mean now! Moved to class level constants. Thanks!